src/python/tritonfrontend/_api/__init__.py

```diff
@@ -37,3 +37,10 @@
     # TRITON_ENABLE_GRPC=OFF
     # TritonFrontendGrpc Package was not present
     pass
+
+try:
+    from ._metrics import Metrics
+except ImportError:
+    # TRITON_ENABLE_Metrics=OFF
```

Comment:
Make sure `L0_build_variants` passes

Reply:
Passing pipeline with `L0_build_variants`. The CI Pipeline ID is 19814908.